docs/src/reference/the-traversal.asciidoc

```diff
@@ -1127,6 +1127,31 @@ g.V().repeat(both()).times(3).values('age').min()
 IMPORTANT: `min(local)` determines the min of the current, local object (not the objects in the traversal stream).
 This works for `Collection` and `Number`-type objects. For any other object, a min of `Double.NaN` is returned.
 
+[[optional-step]]
+Optional Step
+~~~~~~~~~~~~~
+
+The `optional()`-step returns the result of the specified traversal if it yields a result else it returns the calling
```

Comment:
please say that it is a `(*map*)-step` ... see the convention I use for other steps in the traversal.asciidoc.